packages/internet-header/src/components/post-internet-breadcrumbs/post-internet-breadcrumbs.tsx

```diff
@@ -125,8 +125,10 @@ export class PostInternetBreadcrumbs {
    */
   setBodyScroll(overlay: IBreadcrumbOverlay) {
     if (this.overlayVisible) {
+      // @ts-ignore
       disableBodyScroll(overlay, { reserveScrollBarGap: true });
     } else {
+      // @ts-ignore
       enableBodyScroll(overlay);
```

Comment:
Not sure, if it used to work or if the types are wrong (rather unlikely). For safety, I put comment instead of changing anything.